lib/auth.js

```diff
@@ -50,6 +50,7 @@ const authClient = new Model({
 
     this._bearerToken = response.access_token;
     apiClient.headers.Authorization = 'Bearer ' + this._bearerToken;
+    JSONAPIClient.headers.Authorization = 'Bearer ' + this._bearerToken;
```

Comment:
`JSONAPICLient` is a class, not an instance of a client. New clients are instantiated with:
```js
const client = new JSONAPIClient(APIRootURL, headers, mixins)
```

You can pass default headers to the constructor, then access them from `client.headers` (I think.)

See the Panoptes API client for an example.
https://github.com/zooniverse/panoptes-javascript-client/blob/87f1d174953ccec7a9f86fbbb5dfd387372905b8/lib/api-client.js#L4-L48

The auth client doesn't instantiate an API client, so you'd have to rewrite it to do that before making this change.

Reply:
oh crud. oops you are right.